### PR TITLE
create translation files (template and german)

### PR DESCRIPTION
**Pull request by rschnapka**
new templates, german translation or even added i18n-functionality alltogether

### CODE REVIEW

File: stock_tracking_prodlot/i18n/de.po [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+#. module: stock_tracking_prodlot
+#: model:ir.model,name:stock_tracking_prodlot.model_stock_production_lot
+msgid "Serial Number"
+msgstr "Seriennummer"
+
+#. module: stock_tracking_prodlot
+#: model:ir.model.fields,field_description:stock_tracking_prodlot.field_stock_production_lot_tracking_id
+#: field:stock.production.lot,tracking_id:0
+msgid "pack"
+msgstr "Packung"
```
Comment (pedrobaeza) [inline stock_tracking_prodlot/i18n/de.po:28]: lower case

File: stock_tracking_state/i18n/stock_tracking_state.pot [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: \n"
+"X-Generator: Poedit 1.5.4\n"
+
+#. module: stock_tracking_state
+#: view:stock.tracking:0
+msgid "{'invisible':[('state', '=', 'close')]}"
```
Comment (pedrobaeza) [inline stock_tracking_state/i18n/stock_tracking_state.pot]: Remove this entry

File: stock_tracking_prodlot/i18n/de.po (hunk 1 above)
Comment (yvaucher) [inline stock_tracking_prodlot/i18n/de.po:28]: @pedrobaeza In german all nouns must be capitalized.

Comment (pedrobaeza) [inline stock_tracking_prodlot/i18n/de.po:28]: Although in the middle of the sentence?

**Comment by yvaucher** [comment]
Errors are unrelated

:+1:

File: stock_tracking_prodlot/i18n/de.po (hunk 1 above)
Comment (rschnapka) [inline stock_tracking_prodlot/i18n/de.po:28]: Yes, german is a bit more complex than other languages in thes respect. As a pack in this context is a box, carton, or container or the like. So this should be a noun as commented already. I would stick with capitalization here...

Comment (pedrobaeza) [inline stock_tracking_prodlot/i18n/de.po:28]: OK, got it. Thanks for the explanation.

**Comment by laetitia-gangloff** [comment]
technically :+1:

**Pull request merged**